style.css

```diff
@@ -1729,3 +1792,1424 @@ ul li {
   text-align: center;
   letter-spacing: 0.03em;
 }
+
+@media screen and (min-width: 768px) {
+  * {
+    margin: 0;
+  }
+
+  body {
+    position: relative;
+    box-sizing: border-box;
+    width: auto;
+    height: 2650px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    background: #f7f7f7;
+    justify-content: center;
+    align-content: center;
+    margin: 0;
+    overflow-x: hidden;
+  }
+
+  .container {
+    position: relative;
+    box-sizing: border-box;
+    width: 1490px;
+    height: 5000px;
+    overflow-x: hidden;
+  }
```

Comment:
As the previous reviewers have suggested, kindly adjust the website so that the width does not go overboard. You seem to have manually assigned large width to several parts of the website and that's part of the problem making it not responsive.

Your design:
![image](https://user-images.githubusercontent.com/20554583/206887221-2a4e9351-a35c-4fc1-8120-589994c397fd.png)

Reply:
i did remove it

Reply:
i have been working on it, not removing, kindly help with  a suggestion sir @emmiiorji